app/src/main/java/com/anysoftkeyboard/ime/AnySoftKeyboardSuggestions.java

```diff
@@ -1095,7 +1095,7 @@ private boolean pickDefaultSuggestion(boolean autoCorrectToPreferred) {
 
         if (!TextUtils.isEmpty(actualWordToOutput)) {
             TextEntryState.acceptedDefault(typedWord);
-            final boolean fixed = !TextUtils.equals(typedWord, actualWordToOutput);
+            boolean fixed = !TextUtils.equals(typedWord, actualWordToOutput);
```

Comment:
As far as I recall, it can't stay here either, for the same reason above. `final boolean fixed` makes `fixed` stay... fixed... as long as the keyboard stays on screen.

Reply:
`final` reinstated with commit 90acd3c3a4b8a7192209ac60665eb239f8fafe2e.